coffee_core/src/nurse/strategy.rs

```diff
@@ -74,3 +74,39 @@ impl Handler for GitRepositoryLocallyAbsentStrategy {
         }
     }
 }
+
+/// Stategy for migration of the repository global directory
+/// to a local directory for each network, see [1]
+///
+/// This is a strategy tht return the list of network that
+/// needs to be migrated to to the new repository configuration.
+///
+/// [1] https://github.com/coffee-tools/coffee/issues/234
```

Comment:
To make the link valid
```suggestion
/// Stategy for migration of the repository global directory
/// to a local directory for each network. See [related issue]
///
/// This is a strategy that returns the list of networks that
/// need to be migrated to to the new repository configuration.
///
/// [issue]: https://github.com/coffee-tools/coffee/issues/234
```